v2/cf-nodejs-logging-support/src/lib/logger/root-logger.ts

```diff
@@ -79,9 +79,49 @@ export default class RootLogger extends Logger {
     }
 
     // legacy methods
-    overrideNetworkField() {
-        //to do: create a config file and override in config
+    overrideNetworkField(field: string, value: string): boolean {
+        if (field == null && typeof field != "string") {
+            return false;
+        }
+        // get field and override config
+        const configField = this.config.getFields([field]);
+
+        // if new field, then add as static field
+        if (configField.length == 0) {
+            this.config.addConfig([
+                {
+                    "fields":
+                        [
+                            {
+                                "name": field,
+                                "source": {
+                                    "type": "static",
+                                    "value": value
+                                },
+                                "output": [
+                                    "req-log"
+                                ]
+                            },
+                        ]
+                }
+            ]);
+            return true;
+        }
+
+        // set static source and override
+        configField[0].source = {
+            "type": "static",
+            "value": value
+        };
+        this.config.addConfig([
+            {
+                "fields":
+                    [configField[0]]
+            }
+        ]);
```

Comment:
Is this part really necessary if the field exists already? From my understanding above code should also work in this case.. However, it does not hurt.

Reply:
If the field already exist then be can not assume the output is only "req-log" because it could also be both "msg-log" and "req-log", right?  My idea was to copy the preexisting configuration to avoid this kind of misassumption and assure we only change the source.

Reply:
You are right. We should not clear remove the msg log output in this case. 👍